datastore/usermanager.go

```diff
@@ -220,6 +220,10 @@ func (db *DB) GetUserByUsername(username string) (User, error) {
 
 // GetUserByAPIKey fetches a single user from database
 func (db *DB) GetUserByAPIKey(apiKey, username string) (User, error) {
+	if len(apiKey) == 0 || len(username) == 0 {
```

Comment:
Worths having these two things in a different checks or that could be a security breach?

Reply:
In two different check, if both user and api-key are omitted then it will fail with the first check. The  you correct that, and it fails with the second check. I'm not sure that it helps from a user standpoint